components/front-page/TrendingMarketsCompact.tsx

```diff
@@ -0,0 +1,82 @@
+import { FullMarketFragment } from "@zeitgeistpm/indexer";
+import { useMarketCmsMetadata } from "lib/hooks/queries/cms/useMarketCmsMetadata";
+import { useMarketsStats } from "lib/hooks/queries/useMarketsStats";
+import { useMarketImage } from "lib/hooks/useMarketImage";
+import { isMarketImageBase64Encoded } from "lib/types/create-market";
+import { MarketOutcome, MarketOutcomes } from "lib/types/markets";
+import { getCurrentPrediction } from "lib/util/assets";
+import { isAbsoluteUrl } from "next/dist/shared/lib/utils";
+import Image from "next/image";
+import Link from "next/link";
+
+const TrendingMarketsCompact = ({
+  markets,
+}: {
+  markets: FullMarketFragment[];
+}) => {
+  const { data: marketsStats } = useMarketsStats(
+    markets.map((m) => m.marketId),
+  );
+
+  return (
+    <div>
+      <div className="flex w-full flex-col divide-y divide-solid overflow-hidden rounded-lg bg-white text-sm">
+        {markets.map((market) => (
+          <TrendingMarketRow key={market.marketId} market={market} />
+        ))}
+      </div>
+    </div>
+  );
+};
+
+const TrendingMarketRow = ({ market }: { market: FullMarketFragment }) => {
+  const { img, marketId, question } = market;
+  const { data: image } = useMarketImage(market, {
+    fallback:
+      img && isAbsoluteUrl(img) && !isMarketImageBase64Encoded(img)
+        ? img
+        : undefined,
+  });
+
+  const { data: cmsMetadata } = useMarketCmsMetadata(marketId);
+
+  const prediction = getCurrentPrediction(market.assets, market);
+
+  return (
+    <Link
+      href={`/markets/${marketId}`}
+      className="flex h-[70px] items-center p-4 hover:bg-[#D4E7F4]"
```

Comment:
is the background colour a one-off or might it be best to add it into tailwind config?

Reply:
Not sure what the plan is, I just default to adding it like this until we get duplicates